config/locales/en/main.yml

```diff
@@ -2036,6 +2036,8 @@ en:
       answer_not_datetime: "was not a valid datetime"
       answer_not_decimal: "was not a valid number"
       answer_not_integer: "was not a valid integer number"
+      answer_too_large: "was not less than or equal to %{maximum}"
+      answer_too_small: "was not greater than or equal to %{minumum}"
```

Comment:
I believe we need to be checking four different cases: >, >=, <, <=. Right now it's just >= and <=. You'll see on the questions table there are the `minstrictly` and `maxstrictly` options. Those enable the < and > cases when they are true. See the old `validate_min_max` for what I mean. Can we add those errors and test coverage for them?